storm-core/src/jvm/backtype/storm/messaging/netty/Context.java

```diff
@@ -55,6 +54,7 @@ public void prepare(Map storm_conf) {
         int maxWorkers = Utils.getInt(storm_conf.get(Config.STORM_MESSAGING_NETTY_CLIENT_WORKER_THREADS));
 		ThreadFactory bossFactory = new NettyRenameThreadFactory("client" + "-boss");
         ThreadFactory workerFactory = new NettyRenameThreadFactory("client" + "-worker");
+        // TODO investigate impact of having one worker
```

Comment:
Do we need to investigate this? If so please either do the investigation.  either way please revert the changes to this file, there are no changes really worth making.